src/test/java/com/SAS/crudoperations/TeamCRUDTest.java

```diff
@@ -61,7 +61,7 @@ void removeOwnerFromTeam() {
 
     @Test
     void addAndRemoveFacilityTeam() {
-        assertTrue(TeamCRUD.addFacilityToTeam("Macabi", "Terner", "STADIUM", "Beer-Sheva"));
+        assertTrue(TeamCRUD.addOrEditFacilityToTeam("Macabi", "Terner", "STADIUM", "Beer-Sheva"));
         assertTrue(TeamCRUD.removeFacilityFromTeam("Macabi", "Terner"));
 
```

Comment:
does add or edit changes an exsiting team?

Reply:
It adds an asset to the team - player / coach or facility.